spec/models/description_form_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+require 'active_model'
+require 'app/models/description_form'
+
+RSpec.describe DescriptionForm do
+  describe 'validations' do
+    it 'is valid without a description' do
+      expect(DescriptionForm.new.valid?).to eq true
+    end
+
+    it 'is valid with a description' do
+      expect(DescriptionForm.new(description: 'my taps are leaking').valid?).to eq true
```

Comment:
How about splitting this into two lines to improve readability? i.e.

```ruby
form = DescriptionForm.new(description: 'my taps are leaking')
expect(form).valid?).to eq true
```

(NB: Because of RSpec's built-in matchers, you can also do `expect(form).to be_valid`)